spec/requests/alert_definitions_spec.rb

```diff
@@ -306,4 +306,24 @@
     expect(alert_definition_profiles.first.reload.description).to eq("Updated Test Alert Profile 1")
     expect(alert_definition_profiles.second.reload.description).to eq("Updated Test Alert Profile 2")
   end
+
+  it "assings an alert definition to a profile" do
+    api_basic_authorize
```

Comment:
It greatly helps with readibility if you can group these steps into each phase of the four-phase test, with an empty line separating each:

```ruby
api_basic_authorize
alert_definition_profile = FactoryGirl.create(:miq_alert_set)
alert_definition = FactoryGirl.create(:miq_alert)

post(api_alert_definition_profile_url(nil, alert_definition_profile) + "/alert_definitions", :params => gen_request(:assign, alert_definition))

expect(response).to have_http_status(:ok)
expect(response.parsed_body["results"].count).to eq(1)
expect(alert_definition_profile.reload.members.first).to eq(alert_definition)
```

Reply:
done